plotjuggler_base/include/PlotJuggler/reactive_function.h

```diff
@@ -25,7 +31,10 @@ struct TimeseriesRef
 
   void set(unsigned index, double x, double y);
 
-  double atTime(double t) const;
+  double atTime(double t, MatchType match_type) const;
```

Comment:
Adding new argument breaks older scripts. The best way it should be a default value but ain't sure it's possible with lua.